images/debian/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RELEASE
+
+FROM debian:${RELEASE}
+
+# udev is needed for booting a "real" VM, setting up the ttyS0 console properly
+# kmod is needed for modprobing modules
+# systemd is needed for running as PID 1 as /sbin/init
+# Also, other utilities are installed
+RUN apt-get update && apt-get install -y \
+      curl \
+      dbus \
+      kmod \
+      iproute2 \
+      iputils-ping \
+      net-tools \
+      openssh-server \
+      sudo \
+      systemd \
+      udev \
+      vim-tiny \
+      wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+# cleanup machine-id
+    echo "" > /etc/machine-id && echo "" > /var/lib/dbus/machine-id && \
```

Comment:
the layers are squashed anyways when using ignite later, so you can put these not-apt statements in its own RUN directive, to avoid redoing the apt operations if the latter change